requirements/dev-requirements.txt

```diff
@@ -15,7 +15,7 @@ fixture
 sniffer
 psutil  # for memory profiling
 django-extensions
-ipython==4.2.1
+ipython==5.2.2
```

Comment:
FYI, I did find some issues with ipdb and ipython5, but it looks like they've been resolved: https://github.com/gotcha/ipdb/pull/108